README.md

```diff
@@ -21,7 +21,15 @@ Currently the following variables are supported:
 
 ### General
 
-* `localectl_var_name` - var\_name description
+* `localectl_system_locale` - Defaults to LANG=en_US.UTF-8. Use this to set the desired language System locale designation for your system.
```

Comment:
Varname should probably `localectl_locale` to correspond to the underlying `localctl set-locale` subcommand.